sdk/daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -123,7 +123,7 @@ damlStart tmpDir disableUpgradeValidation = do
             , "    npm-scope: daml.js"
             , "  java:"
             , "    output-directory: ui/java"
-            ] ++ [ "build-options:\n- --warn-bad-interface-instances=yes\n- --warn-bad-exceptions=yes" | disableUpgradeValidation ]
+            ] ++ [ "build-options:\n- -Wupgrade-interfaces\n- -Wupgrade-exceptions" | disableUpgradeValidation ]
```

Comment:
Given its `disableUpgradeValidation`, should this not fully disable the warnings?

Reply:
Just tried this, yes this works great!